### ISSUE TITLE
SX chips cannot be updated ....

### ISSUE THREAD

**Issue by pegast73** [sthetix/OLED-DIAGRAM#3]
Hi. I need advice. I have SXCore chips that have Spacecraft_nx version 0.1 installed since purchase. I would like to reinstall them at 0.2 or even the new 0.21_hotfix for now, but the installation of the new fw will not start at all via USB. I also bought a GD-link 0B programmer and the situation is the same there. Those chips where there was still fw SX, so they can be reinstalled, the installation will still start. But those who came to me with a rocket do not react at all. Let me remind you that these are originally SX chips and no longer new HWFLY clones. Thanks for your help.

**Comment by sthetix** [comment]
are you confident that your chip is the real sx produc? btw space 0.2.0 is
for oled, not for other variants

**Comment by sthetix** [comment]
The thing is , you want to install this chip to OLED?

**Comment by pegast73** [comment]
No, I haven't even had OLED in my hands yet. The chips will go to V2 Mariko. I know that OLED is being solved here, but I've been watching you for a long time and I don't know who else could help me, I take you as an expert ....

**Comment by sthetix** [comment]
as i said previously, u don't /. never uodate the firmware to 0.2.0

use spacecraft 0.1.0 for v2

**Comment by pegast73** [comment]
In the problematic ones, there is about 0.05 or some beta or modified version. So even if I wanted to flash to 0.1, I still can't. I don't believe that there is the 0.1 in them, which is normally available for download, as they are the original SX chips and also someone had to upgrade them to a rocket before selling to Ali ...

**Comment by sthetix** [comment]
try flashing with the gdlink then.

**Comment by pegast73** [comment]
If I could flash them to public 0.1.0 and verify my theory about a faulty or bluffed fw, I would be better. After all, these are the same chips that I have with the original fw SX, which can be easily changed fw ....

**Comment by pegast73** [comment]
> 

It doesn't work, the ice turns red, it goes out and gdlink doesn't see them, not something to flash.

**Comment by sthetix** [comment]
Gd link should always detect the chip if it doesn't then your chip is not
the original sx core, but the hwfly core.

**Comment by pegast73** [comment]
Depending on the contents of the motherboard and the appearance, it should be the original SX C.

**Comment by sthetix** [comment]
if it is a true sx product you can flash it

**Comment by pegast73** [comment]
I already understand that, clone makers also sell the original SX chips as HWFLY, they put their deliberately modified software there so that they cannot be read and written and therefore not updated. He found an article where you also discussed. You can lock it up, because for me there is no other way than to accept it and tolerate such insidious clones ....

**Comment by sthetix** [comment]
have you tried flashing the clones with RPI? the guide is on gbatemp
